.circleci/config.yml

```diff
@@ -111,20 +111,20 @@ jobs:
 
       - run:
           name: Javadoc
-          command: make ci-javadoc
+          command: make javadoc
       - store_artifacts:
-          path: ci/javadoc/build/reports
+          path: code/edge/build/dokka/javadoc
 
       - run:
           name: Build
-          command: make ci-build
+          command: make assemble-phone
 
       - run:
           name: Build App
-          command: make ci-build-app
+          command: make assemble-app
 
       - android/run-tests:
-          test-command: make ci-unit-test
+          test-command: make unit-test
```

Comment:
Userprofile uses make unit-test-coverage which include running unit test and create coverage. Do you want to include that in this PR or later?

Reply:
It's not clear to me that it is needed to have a separate Makefile target which just runs the unit-test without producing a coverage file. I don't see that User Profile is running their `unit-test` target, only `unit-test-coverage`. I would prefer to keep the target name as `unit-test` while having it produce the coverage files.

Reply:
They have both unit-test and unit-test-coverage.
unit-test can be used for running locally.
unit-test-coverage will run both unit test and unit test coverage.

Reply:
I've updated the Make file to include coverage and non-coverage testing targets so we are consistent with Core team's implementation.